f5router/bigipResources/bigipResources.go

```diff
@@ -97,6 +97,7 @@ type (
 	// Condition for a rule
 	Condition struct {
 		Equals      bool     `json:"equals,omitempty"`
+		BeginsWith  bool     `json:"beginsWith,omitempty"`
```

Comment:
The BIG-IP does use "begins with" in the GUI but this should be `startsWith`:
```
"conditionsReference": {
          "link": "https://localhost/mgmt/tm/ltm/policy/~booch~test-wildcards/rules/0/conditions?ver=12.1.0",
          "isSubcollection": true,
          "items": [
            {
              "kind": "tm:ltm:policy:rules:conditions:conditionsstate",
              "name": "0",
              "fullPath": "0",
              "generation": 949329,
              "selfLink": "https://localhost/mgmt/tm/ltm/policy/~booch~test-wildcards/rules/0/conditions/0?ver=12.1.0",
              "caseInsensitive": true,
              "external": true,
              "host": true,
              "httpHost": true,
              "index": 0,
              "present": true,
              "remote": true,
              "request": true,
              "startsWith": true,
              "values": [
                "services-"
              ]
            }
          ]
        }
```

Reply:
Fixed

Reply:
fixed